embeddings/model/lightning_module/question_answering.py

```diff
@@ -56,6 +57,7 @@ def __init__(
 
         super().__init__(metrics=None, **task_model_kwargs if task_model_kwargs else {})
         self.save_hyperparameters({"downstream_model_type": self.downstream_model_type.__name__})  # type: ignore
+        # without type: ignore error: "Tensor" not callable  [operator]
```

Comment:
Comments should be before `# type:ignore` and add detailed reason for `# type:ignore` e.g. `# type:ignore[misc]`